### ISSUE TITLE
Title: Compress Images for Better Page Speed and User Experience

### ISSUE THREAD

**Issue by Shubham29012001** [varunKT001/tomper-wear-ecommerce#26]
### Describe your issue.

Since I was working on another issue of creating a better footer, I noticed that it takes lots of time to reload the site due to extreme high-quality images on the page. Both of the images on Frontpage take around 9 MB in size. It's a huge size for images as it will hamper the user experience while surfing the site.

We can compress both images so that they will load faster. Could I work on it for GSoC'22?

### Screenshots (if applicable).

![image](https://user-images.githubusercontent.com/59647563/156830557-803376f3-6c2c-4839-b30e-6da0266384d3.png)

**Comment by varunKT001** [comment]
Nice idea @Shubham29012001 👍,
I'll assign you this issue once your previous pull request is merged, as you need to update your fork for this one or you'll face another merge conflict 😕

**Comment by Shubham29012001** [comment]
Hey @varunKT001, can I now work on it since my previous commit has been merged?

**Comment by varunKT001** [comment]
Ya sure 👍

**Comment by varunKT001** [comment]
Also, in your previous pull request, I just noticed that you have done all the changes directly in your main branch. Make sure this time to checkout to a different branch, and make the pull request from that branch.
If you do this, then in case of any mess up, you won't tamper with your main branch. Otherwise, you would need to delete your previous fork and fork it again or perform a hard reset (very painful 🙂)

**Comment by Shubham29012001** [comment]
Hey @varunKT001, I am sorry I just saw your message now. No issue I will create a new branch and add those changes and create a new pull request for it.

**Comment by varunKT001** [comment]
Its okay for now, but try to do it in future 👍
